libs/s25main/Loader.cpp

```diff
@@ -215,7 +215,7 @@ bool Loader::LoadFilesAtStart()
     if(!LoadSounds())
         return false;
 
-    return LoadResources({"io_new", "client", "languages", "logo", "menu", "rttr"});
+    return LoadResources({"io_new", "client", "languages", "logo", "menu", "rttr", "resource_new"});
```

Comment:
please don't call it `new` ("its new until its old") - perhaps use `resource_rttr` instead?

Reply:
We have `io_new` already... Maybe simply put the new icons into that folder?

Reply:
I don't really care either way. Just tell me where to put it … \*pause for inadvertent joke\*

Reply:
Ok, then I make the call and say `io_new` as that contains other icons as well.

Reply:
Alright. FYI, with regards to naming, I followed the original in case that seems odd.

E.g., `closicdn`/`closicsel`/`closicup` = Close Icon Down/Selected/Up

Reply:
Don't mind that but doesn't matter much as we currently refer to images by index rather than by name. Hence we need names such as `014.nx14.ny0.bmp` where it encodes the index and the (draw-)origin

Reply:
Yeah, I could have made use of the origin, but exporting full-size images was easier.